### PR TITLE
Create security_team.md

### PR DESCRIPTION
**Pull request by EThom73**
Who's who in the Security team, what they are responsible for, and how to report an event or incident.

### CODE REVIEW

File: guides/security/security_team_and_reporting.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# The Security Team and reporting incidents
+
+## Who is responsible for Security at Made Tech?
+
+Each team is responsible for ensuring that robust security practices are happening within their own team. This applies to delivery teams (Delivery Lead, Tech Lead, engineers etc.) and to business teams (Sales, People, etc.).
+
+In addition to this, we have a Security Team who are responsible for ensuring secure practices are adopted across the business and that we continue to meet the standards required to acheive our security certifications (ISO 27001, Cyber Essentials +, etc.)
```
Comment (chrisblackburn) [inline guides/security/security_team_and_reporting.md]: do we need the etc.?

**Comment by chrisblackburn** [comment]
I think there are two use-cases here:

- Security team responsibilities and objectives
- How to report and incident

Pondering if it's useful to provide examples of the sort of things to report. e.g. Dan highlighting the Slack data thing was really useful but not OBVIOUS

**Comment by chrisblackburn** [comment]
I don't think my earlier comment was clear. Thoughts on splitting this in to two separate documents? I feel like the use-cases are sufficiently different

File: guides/security/security_team.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# The Security Team and reporting incidents
```
Comment (chrisblackburn) [inline guides/security/security_team.md]: I think this isn't correct now?

Comment (EThom73) [reply guides/security/security_team.md]: Good spot. Thanks

**Pull request closed without merging**